code/__HELPERS/cameras.dm

```diff
@@ -0,0 +1,46 @@
+///Checks if a mob is visible to the cameranet.
+/proc/near_camera(mob/living/mob_to_check)
+	if (!isturf(mob_to_check.loc))
+		return FALSE
+	if(issilicon(mob_to_check))
+		var/mob/living/silicon/silicon_tracked = mob_to_check
+		if((QDELETED(silicon_tracked.builtInCamera) || !silicon_tracked.builtInCamera.can_use()) && !GLOB.cameranet.checkCameraVis(silicon_tracked))
+			return FALSE
+	else if(!GLOB.cameranet.checkCameraVis(mob_to_check))
+		return FALSE
+	return TRUE
+
+/proc/camera_sort(list/L)
```

Comment:
Please document this proc, I don't think it is necessarily obvious what it is doing. 
(It seems like it is only called by `get_camera_list` and will never need to be used anywhere else but it's still good for future people touching this code).
Also if there's a better variable name than `L` then use that.

Reply:
This was just moved over

Reply:
Sure, but you did touch it and you're going to be making more changes in this file anyway 😅